### DISCUSSION TITLE
Adding link to the Default Session on the Splash Screen

### DISCUSSION THREAD

**Discussion by bioinfoMMS** [GMOD/jbrowse-components#2764] [Q&A]
Hello,

I see that on the splash screen there are options to go to an empty session, a linear genome view session, and a SV inspector viewer. Is there way to add other sessions to that list? For instance, I have a default session set up but would like a way to access it from the splash screen. Other than saving it and having it show up in recent sessions, is there another way to do this?

Thanks in advance,
Maddy

**Comment by cmdcolin** [comment]
Hi thanks for posting about this. We considered some ideas related to this here https://github.com/GMOD/jbrowse-components/issues/1384

There is not a great way to do this currently inside jbrowse-web, but definitely want to support stuff like this!

If you wanted to build something right now, perhaps the best way right now would be to have a page external to jbrowse-web that links to pre-defined sessions via the URL bar, but hopefully can get something delivered to help this inside jbrowse-web soon
